tool/coremark/cpe_v1_coremark.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: cpe.cogadvisor.io/v1
+kind: Benchmark
+metadata:
+  name: coremark
+  namespace: default
+spec:
+  benchmarkOperator:
+    name: none
+    namespace: default
+  benchmarkSpec: |
+    parallelism: 1
+    backoffLimit: 0
+    template:
+      spec:
+        containers:
+          - name: coremark
+            image: ghcr.io/sunya-ch/coremark:ubi
```

Comment:
probably we can add the image to `https://recovery.quay.io/organization/sustainable_computing_io`?

Reply:
I do not have a secret key to push the image. Please feel free to pull image from `ghcr.io/sunya-ch` and push to your repo.

Reply:
do you have a dockerfile we can use to build the image?